modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-checkbox/src/main/resources/META-INF/resources/checkbox.soy

```diff
@@ -17,9 +17,9 @@
 	{@param required: bool}
 	{@param showAsSwitcher: bool}
 	{@param showLabel: bool}
-	{@param? tip: string}
-	{@param visible: bool}
 	{@param value: ?}
+	{@param visible: bool}
+	{@param? tip: string}
 
```

Comment:
Hey @alinedoleron you have worked in this file right? Can you amend the formatSource in this file in your other commit? Because we don't want to commit formatSource separated of our changes.

Reply:
Thanks, Clovis!